examples/gaussian_process/plot_compare_gpr_krr.py

```diff
@@ -102,6 +102,7 @@
       % (time.time() - stime))
 
 # Plot results
+plt.figure(figsize = (10,5))
```

Comment:
@RachitKansal : did you check how the front page renders (ie render the docs with a "make html")? Could you post a screenshot here.